docs/source/cli.rst

```diff
@@ -77,3 +80,35 @@ Try it out
                      ),
                  )
      something after code pattern
+
+Tips and Tricks
+---------------
+
+Most of these will make use of
+```xargs`` <http://man7.org/linux/man-pages/man1/xargs.1.html>`_
+
+Using with ``grep``/``git grep`` to find files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    grep -l <search-text> | xargs undebt -p <path-to-pattern>
```

Comment:
Yeah, I think I pushed a commit that marked the diff as old before I had a chance to read this. Fixing.